samples/besu/simpleasset/app/AssetExchangeERC20.js

```diff
@@ -0,0 +1,308 @@
+const {assertEqualBN} = require('./helper/assert')
+const {
+  bufToStr,
+  htlcERC20ArrayToObj,
+  isSha256Hash,
+  newSecretHashPair,
+  nowSeconds,
+  random32,
+  txContractId,
+  txLoggedArgs,
+} = require('./helper/utils')
+const promisify = require('util').promisify
+const sleep = promisify(require('timers').setTimeout)
+const truffleAssert = require('truffle-assertions')
+const fs = require('fs')
+const path = require('path')
+const contract = require("@truffle/contract");
+const Web3 = require ("web3")
+
+const networkHost1 = "localhost"
+const networkPort1 = "8545"
+const networkHost2 = "localhost"
+const networkPort2 = "9544"
+
+
+const AliceERC20TokenContract = require('../build/contracts/AliceERC20.json')
+const BobERC20TokenContract = require('../build/contracts/BobERC20.json')
+
+let htlc1
+let htlc2
+let AliceERC20
+let BobERC20
+let hashPair // shared b/w the two swap contracts in both directions
+
+
+// Function to obtain an instance of a smart contract when provided 
+// as input the path to its JSON file output by 'truffle compile'.
+// (Assuming that the current folder is a sibling folder to build, 
+// the path to json would be ../build/contracts/ContractName.json)
+async function getContractInstance(provider, pathToJson){
+
+	const jsonFile = path.resolve(__dirname, pathToJson)
+	var jsonFileContents = fs.readFileSync(jsonFile)
+	var contractName = contract(JSON.parse(jsonFileContents))
+	contractName.setProvider(provider)
+	var instance = await contractName.deployed()
+
+	return instance
+}
+
+
+// Initialization of the parameters
+async function init(provider1, provider2, contractOwner1, contractOwner2, Alice1, Bob2, tokenSupply, senderInitialBalance) {
+	
+    	htlc1 = await getContractInstance(provider1,'../build/contracts/InteroperationBaseClassERC20.json') 
+    	htlc2 = await getContractInstance(provider2,'../build/contracts/InteroperationBaseClassERC20.json') 
+
+	AliceERC20 = await getContractInstance(provider1, '../build/contracts/AliceERC20.json')
+	
+	BobERC20 = await getContractInstance(provider2, '../build/contracts/BobERC20.json')
+
+// ALREADY RUN ONCE: So commenting it out for now 
+    await AliceERC20.transfer(Alice1, senderInitialBalance, {from: contractOwner1}) // so Alice has some tokens to trade in Network 1
+    await BobERC20.transfer(Bob2, senderInitialBalance, {from: contractOwner2}) // so Bob has some tokens to trade in Network 2
+//*/
+
+// Asserts have to be rewritten
```

Comment:
Do you anticipate using the below code after suitable modifications? If not, just get rid of it.